build.gradle

```diff
@@ -1,14 +1,20 @@
 buildscript {
+    ext.kotlin_version = '1.4.0'
```

Comment:
This version probably shouldn't differ from the `plugins` `id` version we use.

Reply:
Removed. Updated versions of both kotlin plugins from `1.3.72` to `1.4.0`